muxpi/nuc/ethup.sh

```diff
@@ -0,0 +1,18 @@
+ #!/bin/bash
+ DUT_IP="$1"
+ n=1 
+ max=5
+ delay=50
+ while true; do
+    ping -c 1 -W 1 $DUT_IP && break || {
+      if [[ $n -lt $max ]]; then
+        ((n++))
```

Comment:
Here you are updating the count, before the printout, so the printout will be very confusing. Move this to below the echo.

Reply:
True, will fix it